demo/nuxt.config.js

```diff
@@ -13,6 +13,28 @@ export default {
     coreModuleDist
   ],
   router: {
-    middleware: ['routeLoader']
+    middleware: ['auth', 'routeLoader']
+  },
+  auth: {
+    redirect: {
+      login: '/login',
+      logout: '/login',
+      home: '/home',
+      callback: false
+    },
+    strategies: {
+      local: {
+        endpoints: {
+          login: { url: 'https://localhost:8443/login', method: 'post' },
```

Comment:
I think we could use the environment URL for the API endpoint here, even in the demo.

Reply:
Usually what I do in Javascript projects is that I have some kind of proxy `/api => https://localhost:8443`. You want to avoid environment variables on client-side because they should be builded with the project on deployment, as opposed to be available on run. 

Indeed, in production, the front-end is usually beeing served from nginx which can proxy `/api` on whatever he wants.